guides/common/modules/proc_propagating-scap-content-using-puppet-deployment.adoc

```diff
@@ -0,0 +1,35 @@
+[id="Propagating_SCAP_Content_using_Puppet_Deployment_{context}"]
+= Propagating SCAP Content using Puppet Deployment
+
+Using this procedure, you can promote Security Content Automation Protocol (SCAP) content through the load balancer in the scope of the Puppet deployment method.
+
+.Prerequisite
+* Ensure that you have configured {Project} for Puppet deployment of compliance policies.
+For more information, see {ManagingSecurityDocURL}configuring-compliance-policy-deployment-methods_security-compliance[Configuring Compliance Policy Deployment Methods] in _{ManagingSecurityDocTitle}_.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Classes* and click `foreman_scap_client`.
+. Click the *Smart Class Parameter* tab.
+. In the pane to the left of the *Smart Class Parameter* window, click `port`.
+. In the *Default Behavior* area, select the *Override* checkbox.
+. From the *Key Type* list, select `integer`.
+. In the *Default Value* field, enter `{smartproxy_port}`.
+. In the pane to the left of the *Smart Class Parameter* window, click `server`.
+. In the *Default Behavior* area, select the *Override* checkbox.
+. From the *Key Type* list, select `string`.
+. In the *Default Value* field, enter the FQDN of your load balancer, such as `_{loadbalancer-example-com}_`.
+. In the lower left of the *Smart Class Parameter* window, click *Submit*.
+. Continue with deploying a compliance policy using Puppet. 
+For ore information, see:
+* {ManagingSecurityDocURL}Deploying_a_Policy_in_a_Host_Group_Using_Puppet_security-compliance[Deploying a Policy in a Host Group Using Puppet] in _{ManagingSecurityDocTitle}_
+* {ManagingSecurityDocURL}Deploying_a_Policy_on_a_Host_Using_Puppet_security-compliance[Deploying a Policy on a Host Using Puppet] in _{ManagingSecurityDocTitle}_
+
+.Verification
+. On the client, verify that the `/etc/foreman_scap_client/config.yaml` file contains the following lines:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# Foreman proxy to which reports should be uploaded
```

Comment:
```suggestion
# {SmartProxyServer} to which reports should be uploaded
```

Reply:
I wouldn't do this replacement and go with the literal, because we're quoting code.

Reply:
OK, works for me.